src/c++/perf_analyzer/genai-perf/tests/test_llm_inputs.py

```diff
@@ -358,3 +358,55 @@ def test_synthetic_to_openai_completions(self, default_tokenizer):
 
         assert pa_json is not None
         assert len(pa_json["data"]) == 5
+
+    @pytest.mark.parametrize(
+        "output_format",
+        [
+            (OutputFormat.OPENAI_CHAT_COMPLETIONS),
+            (OutputFormat.OPENAI_COMPLETIONS),
+            (OutputFormat.TRTLLM),
+            (OutputFormat.VLLM),
+        ],
+    )
+    def test_llm_inputs_extra_inputs(self, default_tokenizer, output_format) -> None:
+        # Simulate --request-input arguments
+        request_inputs = {"additional_key": "additional_value"}
+
+        # Generate input data with the additional request inputs
+        pa_json = LlmInputs.create_llm_inputs(
+            input_type=PromptSource.SYNTHETIC,
+            output_format=output_format,
+            num_of_output_prompts=5,  # Generate a small number of prompts for the test
+            add_model_name=False,
+            add_stream=True,
+            tokenizer=default_tokenizer,
+            extra_inputs=request_inputs,  # Pass the simulated --request-input arguments here
+        )
+
+        assert len(pa_json["data"]) == 5
+
+        # Verify that each entry in the generated JSON includes the additional key-value pairs
+        if (
+            output_format == OutputFormat.OPENAI_CHAT_COMPLETIONS
+            or output_format == OutputFormat.OPENAI_COMPLETIONS
+        ):
+            for entry in pa_json.get("data", []):
```

Comment:
Since we already checked that `"data"` exists in `pa_json` and it has 5 items in line 386, I think we should not set a default value (since it won't throw error):
```suggestion
            for entry in pa_json["data"]:
```